tidb-server/main.go

```diff
@@ -589,6 +589,7 @@ func setGlobalVars() {
 
 	atomic.StoreUint64(&tikv.CommitMaxBackoff, uint64(parseDuration(cfg.TiKVClient.CommitTimeout).Seconds()*1000))
 	tikv.RegionCacheTTLSec = int64(cfg.TiKVClient.RegionCacheTTL)
+	tikv.EnableForwarding = cfg.EnableForwarding
```

Comment:
Is it a global config? That is to say, it's shared with PD.

Reply:
Yes. It's introduced in https://github.com/pingcap/tidb/pull/23025 .